nextflow_schema.json

```
@@ -234,7 +234,7 @@
                 },
                 "max_time": {
                     "type": "string",
-                    "default": "2d",
+                    "default": "240 h",
```

```suggestion
                    "default": "240.h",
```
as per: https://github.com/KwanLab/Autometa/blob/f00e67de2869d4ac90226e8a15cabbc7c427b7d3/nextflow.config#L108